WordPress/src/main/res/values/strings.xml

```diff
@@ -236,6 +236,7 @@
     <!-- buttons on post cards -->
     <string name="button_edit">Edit</string>
     <string name="button_publish">Publish</string>
+    <string name="button_save">Save</string>
```

Comment:
We can save a translation here by using `@string/save` and adding `translatable="false"` (like we do for `button_delete` and `button_more`).